fetch_artist_genre.py

```
@@ -0,0 +1,151 @@
+            self.all_tags.append(data.strip())
+
+    def handle_endtag(self, tag):
+        if tag == 'a':
+            self.in_tag = False
+
+    def get_genres(self):
+        return [t for t in self.all_tags if t.lower() not in self.LOCATION_TAGS]
+
+
+def fetch_bandcamp_genres(url):
+    req = urllib.request.Request(url, headers={'User-Agent': 'Mozilla/5.0'})
```

Ruff (0.16.1)

[error] Audit URL open for permitted schemes. Allowing use of `file:` or custom schemes is often unexpected.

(S310)